go.mod

```diff
@@ -32,7 +32,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.11.0
 	github.com/robfig/cron/v3 v3.0.1
-	github.com/sigstore/cosign v1.3.0
+	github.com/sigstore/cosign v1.5.0
```

Comment:
I think cosign 1.5.0 is being worked on here: https://github.com/kyverno/kyverno/pull/3089. I don't think you need it bump it here for these changes.

Reply:
ah great! 🤦🏻‍♂️

Reply:
but I need some of the methods that came with cosign 1.5.0

Reply:
@JimBugwadia, maybe you can add us both to your fork repository to let us be able to send a code over there to work in collab with your Draft PR. 🙈

Reply:
Hi @developer-guy - sure! I've added both of you.